src/libs/ReportUtils.js

```diff
@@ -865,8 +865,9 @@ function buildOptimisticChatReport(
  * @returns {Object}
  */
 function buildOptimisticCreatedReportAction(ownerEmail) {
+    const reportActionID = NumberUtils.rand64();
     return {
-        0: {
+        [reportActionID]: {
```

Comment:
Are you sure this change doesn't break other commands calling this function?

Reply:
@thienlnam we should remove this change and make it later in https://github.com/Expensify/App/pull/13118 when we switch all the reportActions data to be keyed by reportActionID

Reply:
ah shiet okay sounds good